packages/rollup-tests/src/ignored-tests.js

```diff
@@ -27,8 +28,10 @@ const ignoreTests = [
   'rollup@function@preserve-symlink: follows symlinks',
   'rollup@function@symlink: follows symlinks',
   // The rolldown output chunk including `module comment` caused line offset, the rollup provider the fake sourcemap can't remapping.
-  // cSpell:disable
   "rollup@sourcemaps@render-chunk-babili: generates valid sourcemap when source could not be determined@generates es",
+  // Here has unexpected error `Error: nul byte found in provided data at position: 0` from rust due to #967.
+  // It crashed at call `banner` function at rust. 
+  "rollup@sourcemaps@excludes-plugin-helpers: excludes plugin helpers from sources@generates es",
```

Comment:
Here the error is wired, cc @Brooooooklyn  could you meet it ?

Reply:
What's the error message here?

Reply:
<img width="1043" alt="image" src="https://github.com/rolldown/rolldown/assets/14008915/4d6cbc94-70ea-4a9a-90b5-f1daa6df3c9b">